lib/constants/constants.go

```diff
@@ -204,6 +204,9 @@ const (
 	// If not empty, turns the preflight checks off
 	PreflightChecksOffEnvVar = "GRAVITY_CHECKS_OFF"
 
+	// BlockingOperationEnvVar specifies whether to wait for operation to complete
+	BlockingOperationEnvVar = "GRAVITY_BLOCKING_OPERATION"
```

Comment:
This is an intermediate step to make robotest happy.
The idea is forcing all cluster operations to execute in a (systemd) service with the frontend querying the operation progress. It is kind of already implemented for cluster update and needs to be implemented for other cluster operations.